### PR TITLE
feat: show onchain balance in increase spending capacity screen

### LINKED ISSUE #282: Show current onchain balance on "Increase Spending Balance" page
This would make it easier to know how much you can open a channel without having to deposit more sats.

image

### PR DESCRIPTION
**Pull request by im-adithya**
Fixes #282

Screenshot 2024-07-22 at 7 01 21 PM

Done similar to extension:

Screenshot 2024-07-22 at 7 02 26 PM

### CODE REVIEW

File: frontend/src/screens/channels/IncreaseOutgoingCapacity.tsx [hunk 1]
```diff
@@ -241,6 +242,10 @@ function NewChannelInternal({ network }: { network: Network }) {
               setAmount(e.target.value.trim());
             }}
           />
+          <div className="text-muted-foreground text-sm">
+            Balance:{" "}
```
Comment (rolznz) [inline frontend/src/screens/channels/IncreaseOutgoingCapacity.tsx:246]:
```suggestion
            Current savings balance: 
```

Comment (im-adithya) [reply frontend/src/screens/channels/IncreaseOutgoingCapacity.tsx]: Done!

**Review by rolznz — APPROVED** [review]
tACK

File: frontend/src/screens/channels/IncreaseOutgoingCapacity.tsx [hunk 2]
```diff
@@ -126,7 +126,9 @@ function NewChannelInternal({ network }: { network: Network }) {
     }
   }, [order.paymentMethod, selectedPeer]);
 
-  const [showAdvanced, setShowAdvanced] = React.useState(false);
+  const [showAdvanced, setShowAdvanced] = React.useState(
```
[the remark this answers is not in the document]
Comment (im-adithya) [reply frontend/src/screens/channels/IncreaseOutgoingCapacity.tsx]: But we won't show the page till the suggestions are loaded (we show Loading component), that's why I did this

[the remark this answers is not in the document]
Comment (rolznz) [reply frontend/src/screens/channels/IncreaseOutgoingCapacity.tsx]: @im-adithya your change applies to the initial state of the variable, which means it applies instantly when the data is still loading.

[the remark this answers is not in the document]
Comment (im-adithya) [reply frontend/src/screens/channels/IncreaseOutgoingCapacity.tsx]: Ah okay got it, thanks 🙌

**Pull request merged**